plugins/ltac/tauto.ml

```diff
@@ -242,7 +241,7 @@ let tauto_uniform_unit_flags = {
 }
 
 (* This is the compatibility mode (not used) *)
-let tauto_legacy_flags = {
+let _tauto_legacy_flags = {
```

Comment:
Personally, it is relevant to me to remember how it worked in 8.4. And having it type-checked by ocaml ensures a better correctness than if it were a comment. But if it is cluttering too much for others, I can also live with `git log`.